### DISCUSSION TITLE
Icon Changes

### DISCUSSION THREAD

**Discussion by Zohp23** [LiveHelperChat/livehelperchat#1996] [Q&A] [answered]
Hello, tell me, how can I change the toolbar icons, as well as the message sending icon?

**Comment by remdex** [comment]
Read docs quick develoment guide if that's a question about the widget

**Reply by Zohp23** [reply]
Hello, I have read the documentation. CSS methods cannot change icons in the web chat frame. I assume that the icons are blackened in HTML

**Reply by remdex** [reply]
About what you talk widget or back office panel?

**Reply by Zohp23** [reply]
I'm talking about an embedded widget. I wanted to change the icon (arrow) of sending a message in it.

**Reply by remdex** [reply, marked as answer]
Watch https://www.youtube.com/watch?v=pkkMMkIHqoY
You will have to recompile a font if you want custom one, or just use CSS and override default one.
